### PR TITLE
Fix context without CONFIG_IKCONFIG in kernel

### LINKED ISSUE #1765: `paging_enabled` does not actually require the `requires_debug_syms` decorator.
### Description

image

https://github.com/pwndbg/pwndbg/blob/44d75e3bd6d9741daf87c933275369fcfeaeebe7/pwndbg/gdblib/kernel/__init__.py#L513-L523

https://github.com/pwndbg/pwndbg/blob/44d75e3bd6d9741daf87c933275369fcfeaeebe7/pwndbg/gdblib/kernel/__init__.py#L195-L197

https://github.com/pwndbg/pwndbg/blob/44d75e3bd6d9741daf87c933275369fcfeaeebe7/pwndbg/gdblib/kernel/__init__.py#L364-L366

`paging_enabled` does not actually require the `requires_debug_syms` decorator.

### PR DESCRIPTION
**Pull request by lonnywong**
Fix the issue below:
```
pwndbg> context
Traceback (most recent call last):
  File "/home/lonny/pwndbg/pwndbg/commands/__init__.py", line 184, in __call__
    return self.function(*args, **kwargs)
  File "/home/lonny/pwndbg/pwndbg/commands/__init__.py", line 331, in _OnlyWhenRunning
    return function(*a, **kw)
  File "/home/lonny/pwndbg/pwndbg/commands/context.py", line 399, in context
    func(
  File "/home/lonny/pwndbg/pwndbg/commands/context.py", line 507, in context_regs
    regs = get_regs()
  File "/home/lonny/pwndbg/pwndbg/commands/context.py", line 581, in get_regs
    desc = pwndbg.chain.format(value)
  File "/home/lonny/pwndbg/pwndbg/chain.py", line 125, in format
    rest.append(M.get(link, symbol))
  File "/home/lonny/pwndbg/pwndbg/color/memory.py", line 30, in get
    page = pwndbg.gdblib.vmmap.find(int(address))
  File "/home/lonny/pwndbg/pwndbg/lib/cache.py", line 137, in decorator
    value = func(*a, **kw)
  File "/home/lonny/pwndbg/pwndbg/gdblib/vmmap.py", line 154, in find
    for page in get():
  File "/home/lonny/pwndbg/pwndbg/lib/cache.py", line 137, in decorator
    value = func(*a, **kw)
  File "/home/lonny/pwndbg/pwndbg/gdblib/vmmap.py", line 119, in get
    pages.extend(kernel_vmmap_via_page_tables())
  File "/home/lonny/pwndbg/pwndbg/lib/cache.py", line 137, in decorator
    value = func(*a, **kw)
  File "/home/lonny/pwndbg/pwndbg/gdblib/vmmap.py", line 442, in kernel_vmmap_via_page_tables
    if not pwndbg.gdblib.kernel.paging_enabled():
  File "/home/lonny/pwndbg/pwndbg/gdblib/kernel/__init__.py", line 45, in func
    raise Exception(f"Function {f.__name__} requires CONFIG_IKCONFIG")
Exception: Function paging_enabled requires CONFIG_IKCONFIG
```

### CODE REVIEW

**Comment by disconnect3d** [comment]
@gsingh93 can we test this somehow? should we just merge it?

**Comment by gsingh93** [comment]
Hmm, I think we can just remove the decorator completely here. All of the `paging_enabled` methods don't require debug syms. cc: @theguy147 in case I'm missing something

File: pwndbg/gdblib/kernel/__init__.py [hunk 1]
```diff
@@ -510,7 +510,7 @@ def virt_to_pfn(virt: int) -> int:
         raise NotImplementedError()
 
 
-@requires_debug_syms()
+@requires_debug_syms(default=False)
```
Comment (disconnect3d) [inline pwndbg/gdblib/kernel/__init__.py]:
```suggestion
```

**Comment by disconnect3d** [comment]
@gsingh93 I am going to go ahead and just remove that decorator here.

**Pull request merged**